iwyu_include_picker.cc

```diff
@@ -1303,15 +1303,15 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepath(
 vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
-  const string quoted_includer = ConvertToQuotedInclude(including_filepath);
-  const string quoted_includee = ConvertToQuotedInclude(included_filepath);
+  const string quoted_includer = ConvertToQuotedInclude(including_filepath, MakeAbsolutePath(GetParentPath(including_filepath)));
```

Comment:
It might... But this may break user-defined mappings for that file. I can remove it and we wait for a bug report or open an issue to check that. Not sure if the mappings work with different relative paths.